config/cable.yml

```diff
@@ -7,4 +7,4 @@ test:
 production:
   adapter: redis
   url: <%= ENV.fetch("REDIS_URL") { "redis://localhost:6379/1" } %>
-  channel_prefix: website_resume_production
+  channel_prefix: npcme_production
```

Comment:
Just another typo from the template I used to build this app haha.  Shoulllld be the last remnant of its past life